tests/CMakeLists.txt

```diff
@@ -201,7 +198,11 @@ if(AVIF_ENABLE_FUZZTEST)
         #   FUZZTEST_REPLAY=/path/to/repro_file.test
         # and running one of the targets below.
         # See https://github.com/google/fuzztest/blob/main/doc/quickstart-cmake.md
-        add_subdirectory(${AVIF_SOURCE_DIR}/ext/fuzztest ${AVIF_SOURCE_DIR}/ext/fuzztest/build.libavif EXCLUDE_FROM_ALL)
+        # Note: The SYSTEM argument for add_subdirectory() is new in CMake
+        # version 3.25. There are compiler warnings in FuzzTest headers. Add
+        # the ext/fuzztest subdirectory with the SYSTEM directory property so
+        # that warnings in its headers are suppressed.
+        add_subdirectory(${AVIF_SOURCE_DIR}/ext/fuzztest ${AVIF_SOURCE_DIR}/ext/fuzztest/build.libavif EXCLUDE_FROM_ALL SYSTEM)
```

Comment:
Note that the use of `SYSTEM` is not guarded by a CMake version check here. This passed our CI workflows. I guess CMake only parses the conditional branches based on the conditions.

Reply:
I was wondering if you wanted to add `cmake_minimum_required(VERSION 3.25)` or check `CMAKE_VERSION` when this was enabled.

Reply:
Do you recommend I add a cmake version check even though this passed our CI workflows?

Reply:
I don't see any workflows that set `AVIF_LOCAL_FUZZTEST`. If an older version of cmake is used and that set, I'd expect at least a warning, but I haven't tested this. Let me check.

Reply:
With 3.21:

```
CMake Error at tests/CMakeLists.txt:205 (add_subdirectory):
  add_subdirectory called with incorrect number of arguments


CMake Error at tests/CMakeLists.txt:212 (fuzztest_setup_fuzzing_flags):
  Unknown CMake command "fuzztest_setup_fuzzing_flags".
```

A cmake notice might be a little more understandable to the user.

Reply:
> I don't see any workflows that set `AVIF_LOCAL_FUZZTEST`.

This is hidden in tests/oss-fuzz/build.sh. It is apparently invoked by cifuzz.yml, but I haven't figured out how it is done.